install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -92,11 +92,12 @@ type WorkspaceConfig struct {
 			IPFSAddr string `json:"ipfsAddr"`
 		} `json:"ipfsCache"`
 		RedisCache struct {
-			Enabled        bool     `json:"enabled"`
-			MasterName     string   `json:"masterName"`
-			SentinelAddrs  []string `json:"sentinelAddrs"`
-			Username       string   `json:"username"`
-			PasswordSecret string   `json:"passwordSecret"`
+			Enabled            bool   `json:"enabled"`
+			SingleHostAddress  string `json:"singleHostAddr"`
+			Username           string `json:"username"`
+			PasswordSecret     string `json:"passwordSecret"`
```

Comment:
I did the code using the `PasswordSecret`. Is this field still required?

Reply:
Yes, we are not removing auth from Redis.

Reply:
Do we have to pass the PasswordSecret here?
https://github.com/gitpod-io/gitpod/blob/f4dc72ffc13a9295a0bc79ccadb49ca708b178a0/install/installer/pkg/components/registry-facade/configmap.go#L40-L45

Reply:
I am sorry, I misunderstood what you meant. The setting [is used here](https://github.com/gitpod-io/gitpod/blob/e34c6d6a285f61aef7edfc7a1bacc9711c93b4d2/install/installer/pkg/components/registry-facade/daemonset.go#L128)